{{cookiecutter.repo_name}}/README.rst

```diff
@@ -0,0 +1,55 @@
+Part of `edX code`__.
+
+__ http://code.edx.org/
+
+{{cookiecutter.project_name}}
+=============================
+
+The ``README.rst`` file, should start with a brief description of the repository
+which sets it in the context of other repositories under the ``edx``
+organization. It should make clear where this fits in to the overall edX
+codebase.
+
+Overview (please modify)
+------------------------
+
+The ``README.rst`` file should then provide an overview of the code in this
+repository, what the main components are and useful entry points for starting
+to understand the code in more detail.
+
+
+Documentation (please modify)
+-----------------------------
+
+The docs for {{cookiecutter.project_name}} is on Read The Docs:  https://{{cookiecutter.repo_name}}.readthedocs.org
```

Comment:
We should just do this on readthedocs.com as it's available now and we won't have to deal with overly painful transition.  just ping me when  merging